src/pages/documentation/adobe-journey-optimizer/release-notes.md

```diff
@@ -9,6 +9,18 @@ keywords:
 
 # Release Notes
 
+## September 28, 2023
+
+### iOS Messaging 4.1.0
+- Added new notifcation tracking API `handleNotificationResponse` with parameters UNNotificationResponse and an optional callback the returns `PushTrackingStatus` enum.
+- The new API will automatically handle the click behaviour (OPENAPP, WEBURL, DEEPLINK) defined for the notification.
+- Deprecated the notifcaiton tracking API `handleNotificationResponse` with parameters UNNotificationResponse, applicationOpened, and customActionId.
+-  Fixed a bug on notification tracking API `handleNotificationResponse` to stop sending tracking hits to edge servers when the notification does not contain tracking information.
+
+### Android Messaging 2.2.0
+- Introduced `MessagingServices` class to automatically build, display, track and handle defined click behaviour on notifications from Adobe Journey Optimizer.
```

Comment:
Same comment here for missing a newline between the header and the bullet point.

Additionally, I prefer to Oxford comma, so "... build, display, track, and handle defined..."